libpages/dns.go

```diff
@@ -48,7 +48,7 @@ func (ErrKeybasePagesRecordTooMany) Error() string {
 	return "more than 1 TXT record are found for " + keybasePagesPrefix
 }
 
-const kbpRecordPrefix = "_keybase_pages."
+const kbpRecordPrefixes = [2]string{"_keybase_pages.", "_keybasepages."}
 
 // LoadRoot loads the root path configured for domain from DNS, with following
```

Comment:
Could you updated this comment block as well to mention there are two possible TXT record prefixes?